projects/plugins/jetpack/modules/search/instant-search/components/dom-event-handler.jsx

```diff
@@ -183,8 +183,12 @@ export default class DomEventHandler extends Component {
 	};
 
 	scrollOverlayToTop = event => {
-		// NOTE: the propertyName need to be aligned with the animation
-		if ( event?.propertyName !== 'opacity' ) {
+		// NOTE: the propertyName need to be aligned with the animation.
+		// We don't scroll the overlay on overlay close.
+		if (
+			event.propertyName === 'opacity' &&
```

Comment:
it should be `===` but not `!==` here, which was a typo

Reply:
Sorry I missed this in the review! 👀

Reply:
@bluefuton nah mate, you are all good. This is my silly typo which shouldn't happen 😄 .